Add Compactor.MaxLookback Option for Limiting Blocks Loaded in Compaction Cycles (#10585)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# smallest-range-oldest-blocks-first, newest-blocks-first.
# CLI flag: -compactor.compaction-jobs-order
[compaction_jobs_order: <string> | default = "smallest-range-oldest-blocks-first"]

> # (experimental) Blocks uploaded before the lookback will not be considered in

Reviewer: ```suggestion
# (experimental) Blocks uploaded before the lookback aren't considered in
```

Reviewer: We avoid using future tense in docs.

# compactor cycles. If set, this value should be larger than all values in

> # `-blocks-storage.tsdb.block-ranges-period`. The value 0s means all blocks will
> # be considered regardless of their upload time.

Reviewer: ```suggestion
# `-blocks-storage.tsdb.block-ranges-period`. A value of 0s means that all blocks
# are considered regardless of their upload time.
```
